src/tool_launcher/tool_launcher.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net452</TargetFramework>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <OutputType>exe</OutputType>
+    <IsPackable>false</IsPackable>
+    <AssemblyName>Microsoft.DotNet.Tools.Launcher</AssemblyName>
+    <Title>dotnet-tool-launcher</Title>
+    <Description>
+      A simple Windows-only shim for launching new processes.
+    </Description>
+    <!-- In other words, a workaround for issues with cmd.exe. -->
+  </PropertyGroup>
+
+  <!-- This project must not have any package references. It must be a standalone .exe file -->
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Linq" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.DotNet.Cli.Utils\ArgumentEscaper.cs" />
```

Comment:
I think it is better to just inline this method. Or reference Microsoft.DotNet.Cli.Utils. One file bring compiled by different project with different framework is hard to maintain

Reply:
Referencing Microsoft.DotNet.Cli.Utils would require we copy two files, *.exe + Microsoft.DotNet.Cli.Utils.dll. I think it's better to inline.

Instead of having to maintain two copies of the same class, what if I move ArgumentEscaper into a folder like "shared" and add as a Compile Include to both tool_launcher.csproj and Microsoft.DotNet.Cli.Utils.csproj?